memegen/stores/image.py

```diff
@@ -1,16 +1,44 @@
 import os
 
+from flask import current_app as app
 
-class ImageStore:
+from ..extensions import redis
+from ..domain import Image, Text
+
+
+def decode_bytstring_dict(dct):
+    out = {}
+    for k, v in dct.items():
+        out[k.decode()] = v.decode()
+    return out
 
-    LATEST = "latest.jpg"
+
+class ImageStore:
 
     def __init__(self, root):
         self.root = root
 
     @property
     def latest(self):
-        return os.path.join(self.root, self.LATEST)
+        data = redis.hgetall('latest')
+        if not data:
+            raise FileNotFoundError()
+
+        data = decode_bytstring_dict(data)
```

Comment:
Perhaps we don't even need a function?

`data = {k.decode(): v.decode() for k,v in data.items()}`